draft-ietf-oauth-identity-chaining.md

```diff
@@ -306,19 +306,21 @@ This section contains two examples, demonstrating how this specification may be
 
 ## Resource server acting as client
 
-Resources servers may act as clients if the following is true:
+As part of completing a request, a resource server in one domain (Domain A) may need to access a resource server in another domain (Domain B). This requires the resource server in Domain A to obtain an Access Token from an authorization server in Domain B, which may then be presented to the resource server in Domain B. A Resource server may use the flows described in this specification by assuming the role of an OAuth client when attempting to access the resource server in Domain B. Resources servers may act as clients if the following is true:
 
 * Authorization Server B is reachable by the resource server by network and is able to perform the appropriate client authentication (if required).
 * The resource server has the ability to determine the authorization server of the protected resource outside its trust domain.
 
 The flow would look like this:
 
 ~~~
-+-------------+          +--------+         +-------------+ +---------+
-|Authorization|          |Resource|         |Authorization| |Protected|
-|Server       |          |Server  |         |Server       | |Resource |
-|Domain A     |          |Domain A|         |Domain B     | |Domain B |
-+-------------+          +--------+         +-------------+ +---------+
++-------------+         +---------+         +-------------+ +---------+
+|Authorization|         |Resource |         |Authorization| |Protected|
+|Server       |         |Server   |         |Server       | |Resource |
+|Domain A     |         |Domain A |         |Domain B     | |Domain B |
+|             |         |acting as|         |             | |         |
+|             |         |Client   |         |             | |         |
++-------------+         +---------+         +-------------+ +---------+
```

Comment:
Can we make the box containing 'RS Domain A acting as client' wider so it's not so tall?

Reply:
Update made/box widened.